vaadin-combo-box-flow/src/test/java/com/vaadin/flow/component/combobox/ComboBoxTest.java

```diff
@@ -283,11 +283,12 @@ public void setValueNull_selectedItemNull() {
     }
 
```

Comment:
Are the changes to `setDataProvider` regarding the data communicator initialization and `enableFetch` covered by unit tests ? At least not in this class

Reply:
No, they were missed, thanks. `LazyLoadingIT` contains the test for verifying that the data provider is not queried eagerly if of backend type, but it's not enough. I've added the unit tests.